cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -16,40 +16,37 @@
 
    ========================================================================== #}
 
-{% macro render(block, is_half_width=false, hide_header_slug=false) %}
+{% macro render(posts, block, is_half_width=false, hide_header_slug=false) %}
     {% import 'macros/category-icon.html' as category_icon %}
```

Comment:
I think you need to update the other places that call `related_posts.render(...)` or set posts to None